.dockerignore

```diff
@@ -0,0 +1,8 @@
+*
```

Comment:
I like this pattern for dockerignore; ignore everything except specifically stated files. Saves mistakenly copying over coredumps or nasty things like that by mistake.